hackathon_site/dashboard/frontend/src/components/cart/CartCard/CartCard.test.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import CartCard from "./CartCard";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import { fireEvent } from "@testing-library/react";
+
+describe("CartCard Item test", () => {
+    const image = "https://i.imgur.com/IO6e5a6.jpg";
+    const title = "Some very long title and stuff";
+    const currentStock = 9;
+    const handleRemove = jest.fn();
```

Comment:
You have to be careful about defining spys outside of the particular test method, because the mock object will never be reset as-is. When you need to have a mock or spy, it's best to define it directly in the test that uses it. For the rest of the tests, you could leave this as a dumb function like `() => {}`.